lib/internal/Magento/Framework/HTTP/PhpEnvironment/RemoteAddress.php

```diff
@@ -64,6 +64,11 @@ public function getRemoteAddress($ipToLong = false)
             return false;
         }
 
+        if (strpos($this->remoteAddress, ',') !== false) {
```

Comment:
You don't need this check. `explode(',', 'abc')` will give you an array nonetheless.